### PR TITLE
[Arista 7050cx3] add dummy MMU configurations for Arista-7050CX3-32S-C32

### PR DESCRIPTION
**Pull request by yxieca**
**- Why I did it**
On Arista-7050CX3-32S-C32, there are constant stream of errors like following.

Nov  3 21:56:24.415190 str2-7050cx3-acs-06 NOTICE swss#orchagent: :- registerInWdDb: No lossless TC found on port Ethernet68

Which causes:
1. loganalyzer to claim test failed.
2. leaving the system without MMU configuration. Which couldn't be good for any IO test.

**- How I did it**
Added these MMU configuraions are copied from another platform and guaranteed to be incorrect for hwsku Arista-7050CX3-32S-C32.

Adding them so that we have A MMU configuration and the system won't throw a whole bunch of errors and leave MMU unconfigured. The correct MMU configuration will come later.

This configuration is definitely not suitable for testing system performance or QoS behavior.

Signed-off-by: Ying Xie <ying.xie@microsoft.com>

**- How to verify it**
Test will have  chance to pass, e.g.:
pc/test_lag_2.py::test_lag[str2-7050cx3-acs-06|PortChannel0001-single_lag] PASSED                                                                                                  [  4%]
pc/test_lag_2.py::test_lag[str2-7050cx3-acs-06|PortChannel0001-lacp_rate] PASSED                                                                                                   [  8%]
pc/test_lag_2.py::test_lag[str2-7050cx3-acs-06|PortChannel0001-fallback] SKIPPED                                                                                                   [ 12%]
pc/test_lag_2.py::test_lag[str2-7050cx3-acs-06|PortChannel0003-single_lag] PASSED                                                                                                  [ 16%]

### CODE REVIEW

File: device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t0.j2 [hunk 1]
```diff
@@ -0,0 +1,54 @@
+
+{%- set default_cable = '300m' %}
+
+{%- set ports2cable = {
+        'torrouter_server'       : '300m',
```
Comment (lguohan) [inline device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t0.j2:5]: is this intentional?

Comment (yxieca) [reply device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t0.j2]: Good catch. This is problem copied from source file. I'll create another PR to address the other device.

File: device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t0.j2 [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    {# Generate list of ports #}
+    {%- for port_idx in range(0,64) %}
+        {%- if PORT_ALL.append("Ethernet%d" % (port_idx * 4)) %}{%- endif %}
+    {%- endfor %}
+{%- endmacro %}
+
+{%- macro generate_buffer_pool_and_profiles() %}
+    "BUFFER_POOL": {
+        "ingress_lossless_pool": {
+            "size": "10875072",
```
Comment (theasianpianist) [inline device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t0.j2:21]: Just out of curiosity, where do we get these numbers from?

Comment (yxieca) [reply device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t0.j2:21]: For accurate numbers, Broadcom :-)

File: device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t1.j2 [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+{%- set ports2cable = {
+        'torrouter_server'       : '5mm',
+        'leafrouter_torrouter'   : '40m',
+        'spinerouter_leafrouter' : '300m'
+        }
+-%}
+
+{%- macro generate_port_lists(PORT_ALL) %}
+    {# Generate list of ports #}
+    {%- for port_idx in range(0,64) %}
```
Comment (neethajohn) [inline device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t1.j2:13]: range should be 0,32

File: device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t0.j2 [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+{%- set ports2cable = {
+        'torrouter_server'       : '5m',
+        'leafrouter_torrouter'   : '40m',
+        'spinerouter_leafrouter' : '300m'
+        }
+-%}
+
+{%- macro generate_port_lists(PORT_ALL) %}
+    {# Generate list of ports #}
+    {%- for port_idx in range(0,64) %}
```
Comment (neethajohn) [inline device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t0.j2:13]: range should be 0,32

Comment (yxieca) [reply device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t0.j2]: Good catch. Thanks.

File: device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t1.j2 (hunk 3 above)
Comment (yxieca) [reply device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t1.j2]: Good catch. Thanks.

File: device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t1.j2 [hunk 5]
```diff
@@ -0,0 +1,54 @@
+
+{%- set default_cable = '300m' %}
+
+{%- set ports2cable = {
+        'torrouter_server'       : '5mm',
```
[the remark this answers is not in the document]
Comment (yxieca) [reply device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t1.j2]: Good catch. Thanks.

**Review by neethajohn — APPROVED** [review]

**Review by theasianpianist — APPROVED** [review]

**Comment by lguohan** [comment]
retest vsimage please

**Comment by yxieca** [comment]
retest vsimage please

**Comment by lguohan** [comment]
retest vsimage please

**Comment by yxieca** [comment]
retest vsimage please

**Pull request merged**